app/src/main/res/layout/fragment_destination_library.xml

```diff
@@ -46,21 +47,23 @@
       app:layout_behavior="@string/appbar_scrolling_view_behavior">
 
       <androidx.recyclerview.widget.RecyclerView
-        tools:listitem="@layout/item_book"
         android:id="@+id/zimfilelist"
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        tools:listitem="@layout/item_book" />
 
     </androidx.swiperefreshlayout.widget.SwipeRefreshLayout>
 
   </org.kiwix.kiwixmobile.core.utils.NestedCoordinatorLayout>
 
-
   <TextView
     android:id="@+id/file_management_no_files"
     style="@style/no_list_content_text"
     android:layout_gravity="center"
+    android:layout_marginBottom="@dimen/material_design_appbar_size"
```

Comment:
this is strange, I don't see this in `fragment_reader.xml` also its a name leaves something to be desired but seeing as Reader renders fine I am guessing it can just be deleted

Reply:
I have added the margin Bottom so that the text/message is placed exactly at the centre just like all other messages and thereby improving the user experience.